psd-web/app/decorators/user_decorator.rb

```diff
@@ -8,11 +8,24 @@ def assignee_short_name(viewing_user:)
     name
   end
 
-  def name
-    if deleted?
-      "#{object.name} [user deleted]"
-    else
-      object.name
-    end
+  def full_name
+    object.name + deleted_suffix
```

Comment:
```suggestion
    name + deleted_suffix
```
Should this just be `name`?

Reply:
good catch. `object.name` was useful when this method was called `name` but now is unneeded.